master/internal/command/tensorboard_manager.go

```diff
@@ -65,7 +76,28 @@ func (t *tensorboardManager) Receive(ctx *actor.Context) error {
 
 	case echo.Context:
 		t.handleAPIRequest(ctx, msg)
+	case tensorboardTick:
+		services := ctx.Ask(t.proxyRef, proxy.GetSummary{}).Get().(map[string]proxy.Service)
+		for _, v := range ctx.Children() {
```

Comment:
non-blocking: personal preference for using a more descriptive variable than `v` for readability.